cclient/response/get_neighbors.c

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 IOTA Foundation
+
+#include "get_neighbors.h"
+
+void neighbor_t_copy(void* _dst, const void* _src) {
+  neighbor_t *dst = (neighbor_t*)_dst, *src = (neighbor_t*)_src;
```

Comment:
i think you don't need the copy function for `utarray` since you are doing a trivial bytes copy

Reply:
the default implementation of ctor_f in utarray is memcpy, but it is more efficient for the neighbor_t structure.

Reply:
not that it is a difference, but how is it better then memcpy?

Reply:
compare to memcpy, the pointer assignment is faster and reduce memory usage.

Reply:
ok then